README.md

```diff
@@ -1 +1,28 @@
 # Weather-App
+
+This project is one of the homeworks for the Code Your Future coding training course. This application is created to let user check the temperature of the city of their desire. It takes users input and fetch data accordingly from the server and displays the fetched data in the browser in a way that is easy to read to the users.
+
+
+# Team members
+
+##Prabesh
```

Comment:
looks that if you dont put a space after `##` it doesn't render properly , you can see it if you click `view` button under `review changes` in `files changed` tab